src/components/organisms/Settings/ProjectList/hooks.ts

```diff
@@ -41,9 +43,17 @@ export default (teamId: string) => {
   }
   const team = teamId ? data?.me?.teams.find(team => team.id === teamId) : data?.me?.myTeam;
 
-  const { data: projectData } = useGetProjectsQuery({
-    variables: { teamId: teamId ?? "", first: 100 },
+  const initProjectPerPage = 5;
+  const projectPerPage = 3;
```

Comment:
Same as mentioned in the Dashboard:
- Move outside hook
- Do we really need to change the amount between initial load and fetchMore? Just doing projectsPerPage = 5 is fine, no?